README.md

```diff
@@ -81,6 +85,8 @@ The 0.x versions of this extension, which used Stylelint 13.x and prior, support
 In current versions of the extension, the extension only supports validating CSS and PostCSS out of the box and requires additional configuration to validate other languages. You will need to:
 
 1. Install the PostCSS syntax for the language you want to validate into your workspace, e.g. [postcss-scss].
+   <!-- prettier-ignore -->
+   <!-- cspell:disable-next-line -->
```

Comment:
@adalinesimonian,

It is also possible to ignore words:

```markdown
<!-- cspell:ignore stylelintcustomsyntax -->
```

Reply:
The spell checker doesn't care where it is placed, so it can go to the bottom of the document, reducing the need for `prettier-ignore`.

Reply:
I originally used `cspell:ignore` but switched it out for `cspell:disable-next-line` once I found out the ignore comment ignores the misspelling for the whole file. I wish there was an ignore comment for a single line just like there is with disable comments; I'm not too fond of having to choose whether or not to ignore all misspellings for a single line or ignore what would otherwise be a misspelling elsewhere for a whole file.

Granted, using a `prettier-ignore` comment to keep a cspell directive from being moved into the wrong place is clunky. I'm not so hardfast in my compromise to change the comments back to an ignore to be unwilling to change it back, but I really do wish I could just ignore certain words/misspellings for specific lines or regions (a la next-line or disable/enable-style).

Reply:
@adalinesimonian,

What would you propose?

Reply:
It'd be neat to have something like

```js
// cspell:ignore-next-line word
// word is valid here
// word is not valid here

// cspell:ignore-next-line word1, word2
// word1 and word2 are valid here
// but not here

// cspell:ignore-start word
// word is valid here
// and here
// cspell:ignore-end word
// but not here
```

Not necessarily the exact same names for the comment directives, but something like that. Would definitely have used it if it was an option. If I wasn't so busy this week trying to get a few things working down here I'd take a crack at a PR if it'd be welcome.